### ISSUE TITLE
Add Launch Details View Controller

### ISSUE THREAD

**Issue by squarefrog** [RocketFanOrg/RocketFanApp#48]
**Is your feature request related to a problem? Please describe.**
You should be able to see all details of a specific launch in a single screen

**Describe the solution you'd like**
A Launch Details view controller, comprising of several smaller child view controllers, each with a specific purpose. For example, `YouTubeViewController` displays either the YouTube thumbnail or a placeholder image if there is no video available.

**Describe alternatives you've considered**

**Additional context**
This should probably consist of the following child view controllers:

- [ ] YouTube viewer
- [ ] Launch Details - Title, Launch Date, Flight Number, Description
- [ ] Flight details - Rocket, Launch Site, Mission ID
- [ ] Picture gallery
- [ ] Payload details - possibly push in another view with the giant table of stats.
- [ ] Links

Below is a mock up of what I've got in mind. After discussion in Slack, we think the payload details can be pushed into another view to make the main view more readable and friendly.

![Single Mission](https://user-images.githubusercontent.com/343450/57100567-2beae300-6d17-11e9-828f-138782716258.png)

**Comment by RJ-Clegg** [comment]
I really like this - design and approach

**Comment by RJ-Clegg** [comment]
I've started putting this view together. I'll build the Rocket details view next week and the for the following views: 

- Core
- Payload
- Orbit 

I think it would work well if we had them in a pager view, like we have with the photos? Then there is no need to break out to different screens and the user can swipe through to different details if they are interested, whilst staying on the same page. Thoughts?

**Comment by squarefrog** [comment]
Yes, I really like that idea. Can surface the data without having to lose the context. Great idea.

**Comment by RJ-Clegg** [comment]
Great, I'll get something going during the course of this week - all if SwiftUI. So it will either be great or a hell. 😅
